astropy/nddata/tests/test_nddata.py

```diff
@@ -15,14 +15,25 @@
 from astropy.nddata.nddata import NDData
 from astropy.nddata.nduncertainty import StdDevUncertainty
 from astropy.utils import NumpyRNGContext
-from astropy.utils.compat.optional_deps import HAS_DASK
+from astropy.utils.compat.optional_deps import HAS_DASK, HAS_PLT
 from astropy.utils.masked import Masked
 from astropy.utils.metadata.tests.test_metadata import MetaBaseTest
 from astropy.wcs import WCS
 from astropy.wcs.wcsapi import BaseHighLevelWCS, HighLevelWCSWrapper, SlicedLowLevelWCS
 
 from .test_nduncertainty import FakeUncertainty
 
+if HAS_PLT:
+    import matplotlib as mpl
+    import matplotlib.pyplot as plt
+
+    from astropy.utils import minversion
+
+    MPL_LT_3_8 = not minversion(mpl, "3.8")
```

Comment:
Even more nitpicky: one can just do `minversion("matplotlib", "3.8")` to avoid the input here. Indeed, I'd move the import of `plt` to the actual test routine.

Reply:
oh nice, I didn't realise that `minversion` accepted module *names* as well !